ray_julia_jll/src/wrappers/any.jl

```diff
@@ -159,7 +159,11 @@ NullPtr(::Type{Buffer}) = BufferFromNull()
 
 FromHex(::Type{ObjectID}, str::AbstractString) = ObjectIDFromHex(str)
 FromRandom(::Type{ObjectID}) = ObjectIDFromRandom()
-Nil(::Type{ObjectID}) = ObjectIDFromNil()
+FromNil(::Type{ObjectID}) = ObjectIDFromNil()
+
+Base.show(io::IO, x::ObjectID) = show(io, "ObjectID: " * Hex(x))
```

Comment:
Note that since `repr` falls back to show it's best to output something that can constructed on the REPL:

>  Note that `repr(x)` is usually similar to how the value of x would be entered in Julia

```suggestion
ObjectID(str::AbstractString) = FromHex(ObjectID, str)

Base.show(io::IO, x::ObjectID) = show(io, "ObjectID(\"$(Hex(x))\")")
```